app/uk/gov/hmrc/vatapi/config/AppContext.scala

```diff
@@ -19,50 +19,43 @@ package uk.gov.hmrc.vatapi.config
 import com.typesafe.config.ConfigFactory
 import javax.inject.{Inject, Singleton}
 import play.api.Play._
-import play.api.{Configuration, Environment}
-import uk.gov.hmrc.play.config.ServicesConfig
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 import uk.gov.hmrc.vatapi.auth.VATAuthEnrolments
 
 @Singleton
-class AppContext @Inject()(
-                            config: Configuration,
-                            environment: Environment
-                          ) extends ServicesConfig with FixedConfig {
+class AppContext @Inject()(config: ServicesConfig) extends FixedConfig {
 
 
   //API Platform Config
-  lazy val appName: String = config.getString("appName").getOrElse(throw new RuntimeException("appName is not configured"))
-  lazy val appUrl: String = config.getString("appUrl").getOrElse(throw new RuntimeException("appUrl is not configured"))
-  lazy val apiGatewayContext: Option[String] = config.getString("api.gateway.context")
-  lazy val apiGatewayRegistrationContext: String = apiGatewayContext.getOrElse(throw new RuntimeException("api.gateway.context is not configured"))
-  lazy val apiGatewayLinkContext: String = apiGatewayContext.map(x => if (x.isEmpty) x else s"/$x").getOrElse("")
-  lazy val registrationEnabled: Boolean = current.configuration.getBoolean(s"$env.microservice.services.service-locator.enabled").getOrElse(true)
-  lazy val serviceLocatorUrl: String = baseUrl("service-locator")
+  lazy val appName: String = config.getString("appName")
+  lazy val appUrl: String = config.getString("appUrl")
+  lazy val apiGatewayContext: String = config.getString("api.gateway.context")
+  lazy val apiGatewayRegistrationContext: String = apiGatewayContext
+  lazy val apiGatewayLinkContext: String = apiGatewayContext
+  lazy val registrationEnabled: Boolean = current.configuration.getBoolean(s"microservice.services.service-locator.enabled").getOrElse(true)
+  lazy val serviceLocatorUrl: String = config.baseUrl("service-locator")
 
   //DES Config
-  lazy val desEnv: String = config.getString(s"$env.microservice.services.des.env").getOrElse(throw new RuntimeException("desEnv is not configured"))
-  lazy val desToken: String = config.getString(s"$env.microservice.services.des.token").getOrElse(throw new RuntimeException("desEnv is not configured"))
-  lazy val desUrl: String = baseUrl("des")
+  lazy val desEnv: String = config.getString(s"microservice.services.des.env")
```

Comment:
There are a few cases where the `env` has been dropped from the config key. Can you inject a `RunMode` and take the env value from there so that these key don't have to change...?

Reply:
As discussed yesterday, we will make changes to the app-config-xxx